CONTRIBUTING.md

```diff
@@ -40,5 +40,54 @@ Python code is expected to follow
 [PEP8](https://www.python.org/dev/peps/pep-0008/) and
 [not commit atrocities](https://www.youtube.com/watch?v=wf-BqAjZb8M).
 JavaScript, CSS/Less, and markup should follow our
-[front-end standards](https://github.com/cfpb/front-end).
+[front-end standards](https://github.com/cfpb/development).
 When in doubt, mimic the styles and patterns in the existing codebase.
+
+### Browser support
+
+We configure our style and script processors to support the following list of browsers.
+
+- Latest 2 releases of all browsers including:
+    - Chrome
+    - Firefox
+    - Safari
+    - Internet Explorer
+    - Edge
+    - Opera
+    - iOS Safari
+    - Opera Mini
+    - Android Browser
+    - BlackBerry Browser
+    - Opera Mobile
+    - Chrome for Android
+    - Firefox for Android
+    - Samsung Internet
+- Internet Explorer 9
```

Comment:
Trivial: whitespace in the list is inconsistent here

Reply:
IE 9 isn't a sub-item of the "Latest 2" item on L:50, it's it's own thang

Reply:
oh ok, makes sense since it's got a version number next to it, i missed that when scanning it.

Reply:
Yep, it's kinda awkward since it's the only one, but I figured it makes the most sense in case we have to peg a version in the future.